README.md

```diff
@@ -320,6 +320,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Andrew Southern](https://github.com/whoamihealay)
   - [Angel Dhakal](https://github.com/angeldhakal)
   - [Anh Tran](https://github.com/longqua69)
+  - [Anietie Akpanumoh](https://github.com/anietieakpanumoh)
   - [Aniesh Kumar](https://github.com/anieshak)
```

Comment:
```suggestion
  - [Aniesh Kumar](https://github.com/anieshak)
  - [Anietie Akpanumoh](https://github.com/anietieakpanumoh)
```

Reply:
Please does this mean I should start all over again?

Reply:
No, you can make this change by editing the file in your fork's branch, or click the commit suggestion button - both work, but I suggest the earlier because you learn from it

Reply:
After editing it on my fork's branch, do I create a PR again?

Reply:
I have applied the suggested change on my fork's branch.